index.html

```diff
@@ -4,15 +4,246 @@
+                    <img src="images/tigroldcoin.png" alt="My third project sample image">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                        <li><a href="#">html</a></li>
+                    </ul>
+                    <button class="see-project" type="button">
+                        <span class="project-title">See Project</span> </button>
+                </div>
+                </div>
+                <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/petros.png" alt="My fourth project sample image">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                        <li><a href="#">html</a></li>
+                    </ul>
+                    <button class="see-project" type="button">
+                        <span class="project-title">See Project</span> </button>
+                </div>
+                </div>
+                <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/geregalta.png" alt="My fifth project sample image">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                        <li><a href="#">html</a></li>
+                    </ul>
+                    <button class="see-project" type="button">
+                        <span class="project-title">See Project</span> </button>
+                </div>
+                </div>
+                <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/hashenge.png" alt="My sixth project sample image">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                        <li><a href="#">html</a></li>
```

Comment:
- [ ] [ OPTIONAL ] For better user experience, these list items (technologies used) should be plain texts and not links since they are only supposed to show a list of the technologies you used without redirect the user. You can consider replacing the `a` tag with `p` tag.